yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1410,12 +1411,20 @@ func getAssessmentIssuesForUnsupportedDatatypes(unsupportedDatatypes []utils.Tab
 			// Coneverting queryissue directly to AssessmentIssue would have lead to the creation of a new function which would have required a lot of cases to be handled and led to code duplication
 			// This converted AssessmentIssue is then appended to the assessmentIssues slice
 			queryissue := queryissue.ReportUnsupportedDatatypes(datatype, colInfo.ColumnName, constants.COLUMN, qualifiedColName)
-			convertedAnalyzeIssue := convertIssueInstanceToAnalyzeIssue(queryissue, "", false, false)
-			issue = convertAnalyzeSchemaIssueToAssessmentIssue(convertedAnalyzeIssue, queryissue.MinimumVersionsFixedIn)
+			fixed, err := queryissue.IsFixedIn(targetDbVersion)
+			if err != nil {
+				log.Warnf("checking if issue %v is supported: %v", queryissue, err)
+			}
+			if !fixed {
+				convertedAnalyzeIssue := convertIssueInstanceToAnalyzeIssue(queryissue, "", false, false)
+				issue = convertAnalyzeSchemaIssueToAssessmentIssue(convertedAnalyzeIssue, queryissue.MinimumVersionsFixedIn)
+				assessmentIssues = append(assessmentIssues, issue)
+			}
```

Comment:
Looks like this is used at multiple places.
Might be worth making this code as a separate function?